### ISSUE TITLE
请教能否设置多个服务器实现负载均衡？

### ISSUE THREAD

**Issue by silly2020** [kuoruan/luci-app-v2ray#129]
看了半天Luci脚本也没弄明白，是不是默认只能配一个服务器？谢谢

**Comment by kuoruan** [comment]
请自行配置，关于 V2Ray 使用问题我们不做讨论
https://toutyrater.github.io/routing/balance2.html

**Comment by silly2020** [comment]
Yes. Thanks so much for prompt response and sorry for posting something not proper.
I was trying to add additional vmess entry in outbound tab but it wasn't effective after reload, nothing changed in /var/etc/v2ray/v2ray.main.json.  I did several tries and removed vmess entries and added back, and no vmess one in json file, seems only entries from original templates work, you can adjust but delete the whole entry.
I removed configuration file and reinstalled Luci to make it worked again, but still extra entry added still is still not effective.
Maybe I did something wrong.

**Comment by kuoruan** [comment]
After adding a outbound, you need to enable it in ```Global Settings``` to make is effective.

**Comment by silly2020** [comment]
Yes. it's what I did.  
Changing parameters, removing line of entry works but adding a new vmess line. 
Something strange is that I could not added back after I removed it. (No effect and the wmess configuration in /var/etc/v2ray/v2ray.main.json is no longer there)  And it effected back after deleted configuration files and installed it again. 
The symptom looks like only entries in template will effect, I'll check again to see if I missed something.
Thanks again.
